action.yml

```diff
@@ -69,7 +69,7 @@ runs:
     - name: Lookup Coverity Build Tool hash
       id: coverity-cache-lookup
       run: |
-        hash=$(curl https://scan.coverity.com/download/${{ inputs.build_language }}/${{ inputs.build_platform }} \
+        hash=$(curl -sSf https://scan.coverity.com/download/${{ inputs.build_language }}/${{ inputs.build_platform }} \
```

Comment:
don't use short options.  no one knows what these mean.  write out the long form so it's easy to understand at a glance.

[`-s` is `--silent`](https://curl.se/docs/manpage.html#-s).  why ?  curl isn't overly verbose, and having some status info in the log is helpful.

[`-S` is `--show-error`](https://curl.se/docs/manpage.html#-S).  this is only needed because you passed in `--silent`.  so drop both of those.

[`-f` is `--fail`.](https://curl.se/docs/manpage.html#-f).  this tells curl to exit non-zero for HTTP/4xx which is indeed what we want.